src/server/processLauncher.ts

```diff
@@ -172,12 +172,9 @@ export async function launchProcess(options: LaunchProcessOptions): Promise<Laun
       options.log(`[pid=${spawnedProcess.pid}] <will force kill>`);
       // Force kill the browser.
       try {
-        if (process.platform === 'win32') {
-          const stdout = childProcess.execSync(`taskkill /pid ${spawnedProcess.pid} /T /F`);
-          options.log(`[pid=${spawnedProcess.pid}] taskkill output: ${stdout.toString()}`);
-        } else {
-          process.kill(-spawnedProcess.pid, 'SIGKILL');
-        }
+        killProcessGroup(spawnedProcess.pid, {
+          onTaskkillExit: (stdout: Buffer): void => options.log(`[pid=${spawnedProcess.pid}] taskkill output: ${stdout.toString()}`)
```

Comment:
I'd just return the output from `killProcessGroup` instead of passing the callback.

Reply:
since its Windows specific, I would keep it separate for now.